src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static Map<String, Integer> fruitsTypeAndAmount = new HashMap<>();
+
+    public static Map<String, Integer> getFruitsTypeAndAmount() {
+        return fruitsTypeAndAmount;
+    }
+
+    public static void setFruitsTypeAndAmount(Map<String, Integer> fruitsTypeAndAmount) {
+        Storage.fruitsTypeAndAmount = fruitsTypeAndAmount;
+    }
```

Comment:
remove set method, instead of get method, provide necessary accessors - put, entryMap etc. Just look where you use get method and create storage methods for these cases